vision_datasets/common/data_manifest.py

```diff
@@ -447,6 +451,17 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
 
         file_reader.close()
 
+        if data_type == DatasetTypes.IMCAP:
+            images_by_id = {}
+            for img in coco_manifest['images']:
+                label = {'image_id': img['id'], 'file_name': img['file_name'], 'caption': []}
+                images_by_id.update({img['id']: ImageDataManifest(img['id'], get_full_sas_or_path(img['file_name']), [], [], label)})
```

Comment:
why pass [] and [] for image width and height?

Reply:
The ground truth file and the prediction file for image caption contain no image width and height.

Reply:
still not justifying passing [] somewhere expecting int....